sycl/include/sycl/ext/oneapi/bfloat16.hpp

```diff
@@ -30,16 +34,17 @@ class bfloat16 {
   bfloat16(const bfloat16 &) = default;
   ~bfloat16() = default;
 
+private:
   // Explicit conversion functions
   static storage_t from_float(const float &a) {
 #if defined(__SYCL_DEVICE_ONLY__)
 #if defined(__NVPTX__)
     return __nvvm_f2bf16_rn(a);
 #else
-    return __spirv_ConvertFToBF16INTEL(a);
+    return __devicelib_ConvertFToBF16INTEL(a);
 #endif
 #else
-    // In case of float value is nan - propagate bfloat16's qnan
+    // In case float value is nan - propagate bfloat16's qnan
     if (std::isnan(a))
       return 0xffc1;
     union {
```

Comment:
A comment to say this code is in sync with the devicelib code could help for future maintenance.